dashboard/src/shared/Config.ts

```diff
@@ -9,6 +9,14 @@ export default class Config {
   public static async getConfig() {
     const url = Config.APIEndpoint;
     const { data } = await axios.get<IConfig>(url);
+
+    // Development environment config overrides
+    // TODO(miguel) Rename env variable to TELEPRESENCE_CONTAINER_NAMESPACE
```

Comment:
Not a big deal but regarding this TODO I wouldn't make the code dependent of telepresence (even if it just a name). I prefer the way it's specified right now.

Reply:
I understand the idea of that comment was to remove the indirection that happens in in the `npm start` but instead either use the out of the box `TELEPRESENCE_CONTAINER_NAMESPACE` or define it in the webpack file. I ignored the later option but let me update the comment.

Thanks!

```
BROWSER=none REACT_APP_KUBEAPPS_NS=${TELEPRESENCE_CONTAINER_NAMESPACE} 
```